analyses/sample-distribution-analysis/01-filter-across-types.R

```diff
@@ -0,0 +1,143 @@
+# This script filters the given dataset to produce a summarized visualization
+# of key variables within the dataset.
+#
+# Chante Bethell for CCDL 2019
+
+# magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Function to filter based on primary_site 
+location_fn <- function(location) {
+  disease_type_vector <- brain_location %>%
```

Comment:
Can you add more documentation to this function? I tend to favor the [old R Google Style Guide recommendations for function documentation](https://web.archive.org/web/20190620164620/https://google.github.io/styleguide/Rguide.xml). Here's an example in the `create-subset-files` analysis: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/create-subset-files/01-create_subset_files.Rmd#L47